File: java/compare.java
import java.util.Scanner;
public class compare {
	public static void main(String[] args) {
		Scanner s = new Scanner(System.in);
		String x = s.next();
		String y = s.next();
		if(x == y){
			System.out.println("Inputs are the same");
		} else { 
			System.out.println("Inputs are not the same");
		}
	}
}

File: go/compare.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	xIn, _ := reader.ReadString('\n')
	xIn = strings.Replace(xIn, "\n", "", 1)
	yIn, _ := reader.ReadString('\n')
	yIn = strings.Replace(yIn, "\n", "", 1)
	if xIn == yIn {
		fmt.Println("Inputs are the same")
	} else {
		fmt.Println("Inputs are not the same")
	}
}
